neurodsp/tests/test_filt_filter.py

```diff
@@ -21,6 +23,18 @@ def test_filter_signal(tsig):
     with raises(ValueError):
         out = filter_signal(tsig, FS, 'bandpass', (8, 12), filter_type='bad')
 
+    # Apply lowpass to low-frequency sine. There should be little attenuation.
```

Comment:
These look like great tests - I just sorta think they are in the wrong place. 

Each test function should test that function specifically. In this case, `filter_signal` is really a pass through function. As set up, these tests test specific FIR filters. So, this means that if this test fails, it would suggest a problem in `filter_signal`, but the actual issue would most likely be in `filter_signal_fir`. So I think these tests should probably be in `test_filter_signal_fir`.

Reply:
Makes sense to me!